eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/index.test.js

```diff
@@ -301,4 +307,104 @@ describe("Grouped Answer Properties", () => {
       );
     });
   });
+
+  describe("Text answers", () => {
+    const ERR_MAX_LENGTH_TOO_LARGE = "ERR_MAX_LENGTH_TOO_LARGE";
+    const ERR_MAX_LENGTH_TOO_SMALL = "ERR_MAX_LENGTH_TOO_SMALL";
+
+    it(`Should render 'Enter a character less than x' error on ${TEXTAREA} answers`, () => {
```

Comment:
could just write textarea instead of using a constant here. I think you would normally use a constant in the description for when you are looping through several tests.